core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7146,12 +7148,46 @@ components:
           $ref: "#/components/schemas/LedgerTransaction"
         receipt:
           $ref: "#/components/schemas/TransactionReceipt"
+        balance_changes:
+          $ref: "#/components/schemas/CommittedTransactionBalanceChanges"
         proposer_timestamp_ms:
           type: integer
           format: int64
           minimum: 0
           maximum: 100000000000000
           description: An integer between `0` and `10^14`, marking the proposer timestamp in ms.
+    CommittedTransactionBalanceChanges:
+      description: |
+        For the given transaction, contains individual entity resource balance changes.
+        The balance changes accounts for the fee payments as well.
+        For failed transactions, current implementation does not return any balance changes (not even the fee payments).
+        This will also change in a future update.
+      type: object
+      required:
+        - fungible_entity_balance_changes
+        - non_fungible_entity_balance_changes
+        - resultant_account_fungible_balances
```

Comment:
I think we could/should probably ditch `resultant_account_fungible_balances` - or move then to a separate opt-in in future - what do you think?

Reply:
Opt-in within opt-in? I'd be hesitant. We'd have to move those `resultant_account_fungible_balances` into its own area. But then again... the idea behind this PR was to reuse LTS models. But maybe we should abandon this and model non-LTS endpoint from scratch? That would result in some OAS type "duplication" I wanted to avoid.

Reply:
I meant a separate opt-in, not an opt-in inside an opt-in.

Let's just remove `resultant_account_fungible_balances` from this model as it's not balance changes, and I can't see the product need right now.

Reply:
Changed description / dropped resultant account balances